pkg/pipelines/scm/github.go

```diff
@@ -53,18 +53,24 @@ func (r *githubSpec) pushBindingName() string {
 
 func (r *githubSpec) prBindingParams() []triggersv1.Param {
 	return []triggersv1.Param{
-		createBindingParam("gitref", "$(body.pull_request.head.ref)"),
-		createBindingParam("gitsha", "$(body.pull_request.head.sha)"),
+		createBindingParam("io.openshift.build.commit.ref", "$(body.pull_request.head.ref)"),
+		createBindingParam("io.openshift.build.commit.id", "$(body.pull_request.head.sha)"),
 		createBindingParam("gitrepositoryurl", "$(body.repository.clone_url)"),
 		createBindingParam("fullname", "$(body.repository.full_name)"),
+		createBindingParam("io.openshift.build.commit.date", "$(body.pull_request.created_at)"),
+		createBindingParam("io.openshift.build.commit.message", "$(body.pull_request.title)"),
+		createBindingParam("io.openshift.build.commit.author", "$(body.pull_request.user.login)"),
 	}
 }
 
 func (r *githubSpec) pushBindingParams() []triggersv1.Param {
 	return []triggersv1.Param{
-		createBindingParam("gitref", "$(body.ref)"),
-		createBindingParam("gitsha", "$(body.head_commit.id)"),
+		createBindingParam("io.openshift.build.commit.ref", "$(body.ref)"),
+		createBindingParam("io.openshift.build.commit.id", "$(body.head_commit.id)"),
```

Comment:
Can we use constants to store label names? The same constants could be reused in GitLab and unit tests.